src/responder/component.ts

```diff
@@ -202,8 +209,11 @@ export class MultiResponderComponent extends Component<ResponderAnchorState, Blo
                     kind: ResponderActionKind.TxMined,
                     identifier: currentItem.identifier,
                     nonce: currentItem.nonce
+                },{
```

Comment:
I think a comment here about why we should check balance after a tx is mined would be appropriate.